medicines/doc-index-updater/src/service_bus_client/mod.rs

```diff
@@ -153,7 +153,7 @@ impl DocIndexUpdaterQueue {
         T: Message,
         RetrievedMessage<T>: ProcessRetrievalError,
     {
-        tracing::info!("Checking for messages");
+        tracing::info!("Checking for messages.");
```

Comment:
Probably best if this was changed to `trace` or `debug` as it's not that informative

Reply:
sounds good